pkg/accounting/common.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package accounting
+
+// data_types
```

Comment:
What are the actual meanings of these constants?

Reply:
trying to create enums for the data types that will be stored in the raw and rollup tables. Static is the data from the pointerdb, bandwith is from bandwith allocation.